src/adapters/philips_hue/lights.rs

```diff
@@ -0,0 +1,146 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
It would be nice to have a `//!` comment in each module explaining what the module is about.